cmd/principal/main.go

```diff
@@ -275,6 +280,9 @@ func NewPrincipalRunCommand() *cobra.Command {
 	command.Flags().BoolVar(&enableResourceProxy, "enable-resource-proxy",
 		env.BoolWithDefault("ARGOCD_PRINCIPAL_ENABLE_RESOURCE_PROXY", true),
 		"Whether to enable the resource proxy")
+	command.Flags().DurationVar(&waitDurationForAgentPing, "wait-duration-for-agent-ping",
+		cmdutil.ParseKeepAlivePingInterval(env.StringWithDefault("WAIT_DURATION_FOR_AGENT_PING", nil, "")),
```

Comment:
I think `WAIT_DURATION_FOR_AGENT_PING` is not a good name for the environment variable. PTAL at how the other environment variables that we use as arguments are named, and make this one align.